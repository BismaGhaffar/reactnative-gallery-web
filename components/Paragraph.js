import styled from 'styled-components'

const Subtitle = styled.p`
  font-size: 16px;
  line-height: 25px;
  @media (max-width: 769px) {
    font-size: 14px;
  }
`

export default Subtitle
