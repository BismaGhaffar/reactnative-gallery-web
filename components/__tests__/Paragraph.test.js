import React from 'react'
import renderer from 'react-test-renderer'
import Paragraph from '../Paragraph'

it('Paragraph can be created', () => {
  const comp = renderer.create(<Paragraph />)
  expect(comp).toBeDefined()
})

it('<Paragraph /> toMatchSnapshot', () => {
  const tree = renderer.create(<Paragraph />).toJSON()
  expect(tree).toMatchSnapshot()
})
