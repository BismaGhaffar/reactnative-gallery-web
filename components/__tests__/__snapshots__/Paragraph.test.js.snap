// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Paragraph /> toMatchSnapshot 1`] = `
<p
  className="Paragraph__Subtitle-s1secd56-0 iVQIYc"
/>
`;
